client/my-sites/plugins/plugin-details-CTA/CTA-button.jsx

```diff
@@ -134,6 +140,15 @@ export default function CTAButton( { plugin, hasEligibilityMessages, disabled }
 			>
 				<EligibilityWarnings
 					currentContext={ 'plugin-details' }
+					title={ translate( 'Upgrade your plan to install plugins' ) }
+					primaryText={
+						eligibilityHolds &&
```

Comment:
Wasn't able to trigger this issue but I can see CTAButton is used without eligibilty props in this premium plugin use case, I guess we should guard the indexOf call? Maybe this needs to be ported to tsx.

https://github.com/Automattic/wp-calypso/blob/4f40f3ff0d18598384d7a6a9c924aa929c5cb6e3/client/my-sites/plugins/plugin-details-CTA/preinstalled-premium-plugins-CTA.jsx#L89

Reply:
I tested it forcing parameters, and it still works as expected with the indexOf call.

Reply:
Thanks!